tests/remote/remote_target.pm

```diff
@@ -1,34 +1,34 @@
 # SUSE's openQA tests
 #
-# Copyright © 2016 SUSE LLC
+# Copyright © 2016-2018 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Remote ssh & vnc installation
-# Maintainer: mkravec <mkravec@suse.com>
+# Summary: Controller/master for remote installations
+# Tags: poo#9576
+# Maintainer: Martin Loviska <mloviska@suse.com>
 
 use base "opensusebasetest";
 use strict;
 use testapi;
 use lockapi;
 use mmapi;
+use mm_network;
 
 # poo#9576
 sub run {
-    # Notice MASTER system is ready for installation
-    assert_screen "remote_slave_ready", 200;
-    mutex_create "installation_ready";
-
-    # Wait until MASTER finishes installing the system
-    wait_for_children;
+    assert_screen("remote_slave_ready", 350);
+    mutex_create("ssh_server_is_available");
```

Comment:
Why did you change the name? I think "installation_ready" is a way better name because it also relates to VNC, "ssh_server" ready less so.

Reply:
Hello Oliver, during debugging I have used more than one mutex which eventually turned out to be redundant. Thank you for your remark! The name has been reverted to "installation_ready".